frontend/packages/console-app/src/components/nodes/NodeStatus.tsx

```diff
@@ -51,7 +53,10 @@ const NodeStatus: React.FC<NodeStatusProps> = ({ node, showPopovers = false, cla
               current={_.startCase(item)}
               consumers={PressureQueries[item](node.metadata.name)}
               humanize={humanizeMap[item]}
-              description={`This node's ${conditionDescriptionMap[item]}. Performance may be degraded.`}
+              description={t(
+                "nodes~This node's {{conditionDescriptionMap}}. Performance may be degraded.",
+                { conditionDescriptionMap: conditionDescriptionMap[item] },
```

Comment:
nit: the variable is not the map, but a condition description.

```suggestion
                "nodes~This node's {{conditionDescription}}. Performance may be degraded.",
                { conditionDescription: conditionDescriptionMap[item] },
```